homeassistant/components/nibe_heatpump/config_flow.py

```diff
@@ -115,7 +115,7 @@ async def async_step_user(
         try:
             info = await validate_input(self.hass, user_input)
         except FieldError as exception:
-            LOGGER.exception("Validation error")
+            LOGGER.info("Validation error %s", exception)
```

Comment:
I'd pick either warning/error or debug if the user doesn't need to see it. Info doesn't seem appropriate.